common/metrics/runtime.go

```diff
@@ -50,6 +86,20 @@ func NewRuntimeMetricsReporter(scope tally.Scope, reportInterval time.Duration,
 		lastNumGC:      memstats.NumGC,
 		quit:           make(chan struct{}),
 	}
+	rReporter.buildInfoScope = scope.Tagged(
+		map[string]string{
+			"revision":      Revision,
```

Comment:
How about we define these tags under other tag definitions under defs.go

Reply:
Done. Defined them in tags.go along with the other tags.